packages/web3-eth-accounts/src/account.ts

```diff
@@ -154,9 +155,13 @@ export const parseAndValidatePrivateKey = (data: Bytes, ignoreLength?: boolean):
  * web3.eth.accounts.hashMessage(web3.utils.utf8ToHex("Hello world")) // Will be hex decoded in hashMessage
  *
  * > "0x8144a6fa26be252b86456491fbcd43c1de7e022241845ffea1c3df066f7cfede"
+ *
+ * web3.eth.accounts.hashMessage("Hello world", true)
```

Comment:
plus also need to update [changelog](https://github.com/web3/web3.js/blob/4.x/packages/web3-eth-accounts/CHANGELOG.md) for this package